include/photo_album.h

```diff
@@ -59,9 +67,12 @@ void Save_PhotoAlbum_Init(PHOTO_ALBUM *photoAlbum);
 u8 PhotoAlbum_GetIndexOfFirstEmptySlot(const PHOTO_ALBUM *photoAlbum);
 u8 PhotoAlbum_GetNumSaved(const PHOTO_ALBUM *photoAlbum);
 void PhotoAlbum_DeletePhotoByIndex(PHOTO_ALBUM *photoAlbum, u8 idx);
-BOOL PhotoAlbum_SetPhotoAtIndex(PHOTO_ALBUM *photoAlbum, const PHOTO *photo, u8 idx);
-BOOL PhotoAlbum_GetPhotoByIndex(const PHOTO_ALBUM *photoAlbum, PHOTO *photo, u8 idx);
-PHOTO *PhotoAlbum_LoadAllInUsePhotos(const PHOTO_ALBUM *photoAlbum, HeapID heapId);
+BOOL PhotoAlbum_SetPhotoAtIndex(PHOTO_ALBUM *photoAlbum, const PHOTO *photo,
+                                u8 idx);
```

Comment:
I don't really like these multi-line breaks over funcs in headers

Reply:
comment is outdated

Reply:
disregard, vscode sucks